File: 1. Model/Voluntario.py


class Voluntario:
    
    def __init__(self, whatsapp, nome, dtNasc, sexo, rg, end, telefone, probl, usr, snh, dataInscricao):
        self.whatsapp      = whatsapp
        self.nome          = nome
        self.dataNasc      = dtNasc
        self.sexo          = sexo
        self.RG            = rg
        self.endereco      = end
        self.telefone      = telefone
        self.problemas     = probl
        self.username      = usr
        self.senha         = snh
        self.dataInscricao = dataInscricao








    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getWhatsapp(self):
        return self.whatsapp

    def getNome(self):
        return self.nome

    def getDataNasc(self):
        return self.dataNasc

    def getSexo(self):
        return self.sexo

    def getRG(self):
        return self.RG

    def getEndereco(self):
        return self.endereco

    def getTelefone(self):
        return self.telefone

    def getProblemas(self):
        return self.problemas

    def getUsername(self):
        return self.username

    def getSenha(self):
        return self.senha
    
    def getDataInscricao(self):
        return self.dataInscricao


    ##SETS
    def setWhatsapp(self, w):
        self.whatsapp = w

    def setNome(self, n):
        self.nome = n

    def setDataNasc(self, d):
        self.dataNasc = d

    def setSexo(self, s):
        self.sexo = s

    def setRG(self, r):
        self.RG = r

    def setEndereco(self, e):
        self.endereco = e

    def setTelefone(self, t):
        self.telefone = t

    def setProblemas(self, p):
        self.problemas = p

    def setUsername(self, u):
        self.username = u

    def setSenha(self, se):
        self.senha = se
    
    def setDataInscricao(self, di):
        self.dataInscricao = di
    
    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Voluntario.java
package model;

import java.util.Date;



public class Voluntario {

    private String nome;
    private String    whatsapp;
    private String    dtNasc;
    private String sexo;
    private String    rg;
    private String end;
    private String    telefone;
    private String probl;
    private String   dataInscricao;



    public Voluntario(String n, String w, String dn, String s, String r, String en, String t, String p, String di) {
        this.nome          = n;
        this.whatsapp      = w;
        this.dtNasc        = dn;
        this.sexo          = s;
        this.rg            = r;
        this.end           = en;
        this.telefone      = t;
        this.probl         = p;
        this.dataInscricao = di;
    }



    // Methods




    // Gets
    public String getWhatsapp() {
        return this.whatsapp;
    }

    public String getNome() {
        return this.nome;
    }

    public String getDtNasc() {
        return this.dtNasc;
    }

    public String getSexo() {
        return this.sexo;
    }

    public String getRg() {
        return this.rg;
    }

    public String getEnd() {
        return this.end;
    }

    public String getTelefone() {
        return this.telefone;
    }

    public String getProbl() {
        return this.probl;
    }

    public String getDataInscricao() {
        return this.dataInscricao;
    }



    // Sets
    public void setWhatsapp(String w) {
        this.whatsapp = w;
    }

    public void setNome(String n) {
        this.nome = n;
    }

    public void setDtData(String dn) {
        this.dtNasc = dn;
    }

    public void setSexo(String s) {
        this.sexo = s;
    }

    public void setRg(String r) {
        this.rg = r;
    }

    public void setEnd(String e) {
        this.end = e;
    }

    public void setTelefone(String t) {
        this.telefone = t;
    }

    public void setProbl(String p) {
        this.probl = p;
    }

    public void setDataInscricao(String di) {
        this.dataInscricao = di;
    }
}
